subsys/bootloader/bl_validation/Kconfig

```diff
@@ -4,13 +4,11 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
-menuconfig SECURE_BOOT_VALIDATION
-	bool
-	prompt "Secure Boot firmware validation"
-	select SECURE_BOOT_CRYPTO
-	select FW_INFO
+menu "Secure Boot firmware validation"
 
-if SECURE_BOOT_VALIDATION
+config SECURE_BOOT_VALIDATION
+	bool "Enable Secure Boot validation code"
+	select FW_INFO
```

Comment:
This is selected from both here and samples/bootloader. Should this be depends?

Reply:
Changed to `depends on`